<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gBox_fileManage">
     <property name="title">
      <string>文件管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7,1">
        <item>
         <widget class="QLabel" name="label_logDir">
          <property name="text">
           <string>日志存放目录</string>
          </property>
          <property name="buddy">
           <cstring>pBtn_LogDir</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineE_logDir"/>
        </item>
        <item>
         <widget class="QPushButton" name="pBtn_LogDir">
          <property name="text">
           <string>选择目录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gBox_tdengine">
     <property name="title">
      <string>TDengine配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7">
          <item>
           <widget class="QLabel" name="label_ipaddr">
            <property name="text">
             <string>IP地址</string>
            </property>
            <property name="buddy">
             <cstring>lineE_ipaddr</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineE_ipaddr"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,7">
          <item>
           <widget class="QLabel" name="label_port">
            <property name="text">
             <string>端口</string>
            </property>
            <property name="buddy">
             <cstring>lineE_port</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineE_port"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,7">
          <item>
           <widget class="QLabel" name="label_dbName">
            <property name="text">
             <string>数据库名</string>
            </property>
            <property name="buddy">
             <cstring>lineE_dbName</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineE_dbName"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gBox_login">
     <property name="title">
      <string>登录配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="cBox_saveUserInfo">
            <property name="text">
             <string>保存用户信息</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,7">
          <item>
           <widget class="QLabel" name="label_userName">
            <property name="text">
             <string>用户名</string>
            </property>
            <property name="buddy">
             <cstring>lineE_userName</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineE_userName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,7">
          <item>
           <widget class="QLabel" name="label_passwd">
            <property name="text">
             <string>密码</string>
            </property>
            <property name="buddy">
             <cstring>lineE_passwd</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineE_passwd"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,1,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
